odoo/models.py

```diff
@@ -4321,20 +4323,23 @@ def _check_company(self, fnames=None):
 
         if inconsistencies:
```

Comment:
the code for company_dependent fields are actually the same as the the regular fields when you refactor `company = self.env.company` in line 4316 too `companies = self.env.company`
we can try to remove the duplicate code